### PR TITLE
Fix test_invalidEscape with recent PyPy3 versions, closes #584

### LINKED ISSUE #584: test_invalidEscape fails with PyPy3
Hi,
I see that https://github.com/PyCQA/pyflakes/pull/354 re-enabled this test for PyPy, however it fails for me with PyPy 7.3.2 (both the 3.6 and 3.7 variants).
```
FAIL: test_invalidEscape (test.test_api.CheckTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/pyflakes/pyflakes/test/test_api.py", line 600, in test_invalidEscape
    sourcePath, [decoding_error])
  File "/tmp/pyflakes/pyflakes/test/test_api.py", line 380, in assertHasErrors
    (count, err.getvalue()), (len(errorList), ''.join(errorList)))
AssertionError: Tuples differ: (1, "[14 chars]3l:1:5: (unicode error) 'unicodeescape' codec [79 chars]^\n") != (1, "[14 chars]3l:1:6: (unicode error) 'unicodeescape' codec [80 chars]^\n")

First differing element 1:
"/tmp/tmp_4l9f33l:1:5: (unicode error) 'unicodeescape' codec [78 chars] ^\n"
"/tmp/tmp_4l9f33l:1:6: (unicode error) 'unicodeescape' codec [79 chars] ^\n"

  (1,
-  "/tmp/tmp_4l9f33l:1:5: (unicode error) 'unicodeescape' codec can't decode "
?                      ^

+  "/tmp/tmp_4l9f33l:1:6: (unicode error) 'unicodeescape' codec can't decode "
?                      ^

   'bytes in position 0-1: truncated \\xXX escape\n'
   "foo = '\\xyz'\n"
-  '    ^\n')
+  '     ^\n')
?   +


----------------------------------------------------------------------
```

### PR DESCRIPTION
**Pull request by sbraz**
@asottile This was surprisingly easy 🙂

I'm not sure if we need to account for older PyPy versions, please let me know.

### CODE REVIEW

**Comment by asottile** [comment]
probably not, could you also update the pypy versions in CI? I believe those are in .appveyor.yml and .travis.yml

**Comment by sbraz** [comment]
Looks like the newer PyPy version changes the way some tests behave with PyPy2. I'm having trouble reading the AppVeyor logs, I am going to use a matrix to make it look more like Travis.

**Comment by asottile** [comment]
the problem with a matrixed approach there is it significantly increases the runtime :(

we should probably just move to github actions if we approach this as a matrix -- would improve the speed as well

**Comment by asottile** [comment]
I can help with that if you want, let me know!

**Comment by sbraz** [comment]
Is it that bad? Travis looked like it was the slowest to run. I'm still pushing this out of curiosity and we can skip this commit later. I haven't used GH actions yet so your help is welcome.

**Comment by sbraz** [comment]
5 minutes vs 2 minutes, it is slower indeed. Also it doesn't work for PyPy2 for some reason. The PyPy3 failure seems to have to do with `\r\n`.

**Comment by asottile** [comment]
1m51s vs 5m7s -- yeah it's quite a bit slower to matricize appveyor unfortunately -- but if it helps you debug we can leave it like that and put it before finishing

**Comment by sbraz** [comment]
Let's hope it passes this time, I have a good feeling about this!

File: pyflakes/test/test_api.py [hunk 1]
```diff
@@ -718,11 +718,7 @@ class IntegrationTests(TestCase):
     Tests of the pyflakes script that actually spawn the script.
     """
 
-    # https://bitbucket.org/pypy/pypy/issues/3069/pypy36-on-windows-incorrect-line-separator
-    if PYPY and sys.version_info >= (3,) and WIN:
-        LINESEP = '\n'
-    else:
-        LINESEP = os.linesep
+    LINESEP = os.linesep
```
Comment (asottile) [inline pyflakes/test/test_api.py:721]: oh nice! forgot about this -- you can probably inline `os.linesep` now -- no need to have the class variable

Comment (sbraz) [reply pyflakes/test/test_api.py]: Done, I'll be back later tonight or tomorrow. Feel free to edit my commits before merging if you need to chane anything.

**Review by asottile — APPROVED** [review]

**Pull request merged**